avocado/utils/memory.py

```diff
@@ -265,8 +267,8 @@ def get_buddy_info(chunk_sizes, nodes="all", zones="all"):
 
     if re.findall("[<>=]", chunk_sizes) and buddy_list:
         size_list = range(len(buddy_list[-1][-1].strip().split()))
-        chunk_sizes = [str(_) for _ in size_list if eval("%s %s" % (_,
-                                                                    chunk_sizes))]
+        chunk_sizes = [str(_) for _ in size_list if eval("%s %s" %
+                                                         (_, chunk_sizes))]
```

Comment:
Ideally this should be in a different patch, though (I can recommend `git cola` or `git citool` for that ;-) )